org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/commons/TextDocumentTest.java

```diff
@@ -155,6 +156,18 @@ public void testEmptyDocumentInc() throws BadLocationException {
 
 	}
 
+	@Test
+	public void testGetLineInformation() throws BadLocationException {
+		Assertions.assertThrows(BadLocationException.class,

```

Comment:
Just another small thing: we tend to do static imports for the JUnit assert methods (see the imports for `assertNotEquals` and `assertNotNull`).